### PR TITLE
Explicit kwargs pass to _get_children method

### PR DESCRIPTION
**Pull request by lakshmi-kannan**
This change https://github.com/StackStorm/st2/pull/2831/files#diff-212b279de225b603d0bc7fe4e00ae449R224 broken the controller because of kwargs abuse in both CLI https://github.com/StackStorm/st2/blob/master/st2client/st2client/utils/httpclient.py#L75 and the controller simply passing the kwargs :/

I fixed controller arguments to explicitly call out what's needed. 

Spun up an instance and hand patched to test the changes

```
vagrant@st2test:~$ st2 run packs.install packs=circle_ci
.................
id: 5797e5e955fc8c2686399ac9
action.ref: packs.install
parameters:
  packs:
  - circle_ci
status: succeeded
start_timestamp: 2016-07-26T22:36:25.183147Z
end_timestamp: 2016-07-26T22:37:00.040937Z
+--------------------------+-------------------------+--------------------------+-------------------------+-------------------------------+
| id                       | status                  | task                     | action                  | start_timestamp               |
+--------------------------+-------------------------+--------------------------+-------------------------+-------------------------------+
| 5797e5e955fc8c25cbd9feaa | succeeded (10s elapsed) | download                 | packs.download          | Tue, 26 Jul 2016 22:36:25 UTC |
| 5797e5f355fc8c25cbd9feac | succeeded (3s elapsed)  | virtualenv_prerun        | packs.virtualenv_prerun | Tue, 26 Jul 2016 22:36:35 UTC |
| 5797e5f755fc8c25cbd9feaf | succeeded (11s elapsed) | setup_virtualenv         | packs.setup_virtualenv  | Tue, 26 Jul 2016 22:36:39 UTC |
| 5797e60355fc8c25cbd9feb1 | succeeded (6s elapsed)  | reload                   | packs.load              | Tue, 26 Jul 2016 22:36:51 UTC |
| 5797e60955fc8c25cbd9feb4 | succeeded (2s elapsed)  | restart-sensor-container | packs.restart_component | Tue, 26 Jul 2016 22:36:57 UTC |
+--------------------------+-------------------------+--------------------------+-------------------------+-------------------------------+
vagrant@st2test:~$
```

TODO:

* [x] Validate UI didn't break

### CODE REVIEW

**Comment by lakshmi-kannan** [comment]
@Kami I fixed the API controller but it looks like the following change caused us to send the token as part of kwargs. I think you should just pop and not do this. I'll leave that fix to you. I already spent a lot of time testing and validating my changes.  https://github.com/StackStorm/st2/pull/2831/files#diff-212b279de225b603d0bc7fe4e00ae449L213

**Comment by DoriftoShoes** [comment]
LGTM

**Comment by Kami** [comment]
@lakshmi-kannan Good catch and thanks.

And yeah, you are right - in get_property both properties should be poped :/

File: st2api/st2api/controllers/v1/actionexecutions.py [hunk 1]
```diff
@@ -213,14 +213,15 @@ def get_one(self, id):
...
+    @jsexpose(arg_types=[str, int, str])
+    def get(self, id, depth=-1, result_fmt=None, **kwargs):
         """
         Retrieve children for the provided action execution.
 
         :rtype: ``list``
         """
-        return self._get_children(id_=id, **kwargs)
+
+        return self._get_children(id_=id, depth=depth, result_fmt=result_fmt)
```
Comment (Kami) [inline st2api/st2api/controllers/v1/actionexecutions.py]: Yeah, that's a lot better anyway :+1:

**Comment by lakshmi-kannan** [comment]
@Kami are you going to make the pop fix?

**Comment by Kami** [comment]
@lakshmi-kannan Yeah, it's already in master.

**Pull request merged**